cs/di-configuration.texy

```diff
@@ -308,6 +286,49 @@ $database = $container->getByType(PDO::class);
 
 
 
+Pokročilá syntaxe
```

Comment:
Nebude lepší to dát před autowiring, protože ten už trošku odbočuje?

Reply:
Muzeme to tam hodit, zase vetsinou pokrocile kapitoly byvaji na konci.

Reply:
To je fakt. Ono by to mělo být na konci kapitoly `Služby` a autowiring by měl být další kapitola, takže mu zvětšit odsazení. `Anonymní služby` by mohly být před autowiringem

Reply:
Presunul jsem, mrknes na to?